src/ui/pages/CredentialDetails/CredentialDetails.scss

```diff
@@ -5,6 +5,21 @@
     }
   }
 
+  .keri-card-template:not(:first-child),
+  .rare-card-template:not(:first-child) {
+    margin: 0;
+    box-shadow: none;
+  }
+
+  .revoked-alert {
+    background-color: #ff7d7a;
```

Comment:
Please do not hardcode color values here and consider adding it to the `color.scss` file instead as a global color variable.